src/modules/dashboard/components/ActionsPage/staticMaps.ts

```diff
@@ -24,6 +37,7 @@ export const EVENT_ROLES_MAP: EventRolesMap = {
   [ColonyAndExtensionsEvents.ColonyFundsMovedBetweenFundingPots]: [
     ColonyRole.Funding,
   ],
+  [ColonyAndExtensionsEvents.TokensMinted]: [ColonyRole.Root],
```

Comment:
Please add the correct message descriptor for this message role type (inside the `ActionsPageEvent` component)

![Screenshot from 2021-01-06 19-32-30](https://user-images.githubusercontent.com/1193222/103801109-f0f59280-5055-11eb-8742-6a21cbd37c73.png)

Also, the generic copy has a typo in it, please fix it.